clang/include/clang/AST/Type.h

```diff
@@ -480,6 +480,12 @@ class Qualifiers {
            // Otherwise in OpenCLC v2.0 s6.5.5: every address space except
            // for __constant can be used as __generic.
            (A == LangAS::opencl_generic && B != LangAS::opencl_constant) ||
+           // We also define global_device and global_host address spaces,
+           // to distinguish global pointers allocated on host to pointers
```

Comment:
I'm not really cool in English grammar, but this comment looks strange
```suggestion
           // to distinguish global pointers allocated on host from pointers
```

Reply:
No need to be modest here - you are absolutely correct.